iOS build fails on Mac M1 with VS2022 using avalonia.xplat template

**Describe the bug**
Using a brand new project created from the Avalonia `xplat` template, on MacOS Monterey, it is not possible to build the iOS project.
**To Reproduce**
Steps to reproduce the behavior:

1. Ensure dotnet core SDK 6.0.300 is installed
2. Ensure the latest version of XCode is installed
3. Create an empty directory
4. Run `dotnet new avalonia.xplat`
5. run `dotnet workload restore`
6. run `dotnet restore`
7. `cd {myproject}.iOS`
8. `dotnet run`
9. See error: 
```
error CS0012: The type 'UIResponder' is defined in an assembly that is not referenced. You must add a reference to assembly 'Xamarin.iOS, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065'.
```

**Expected behavior**
I should be able to build the iOS application and launch it on the Simulator like the Avalonia Wordle application.


**Desktop (please complete the following information):**

- OS: MacOS 
- Version Monterey

**Additional context**
Could not isolate the difference between the Wordle application and the created application from the template. The Avalonia Wordle sample application compiles and launches, but crashes on Startup. Would appreciate a workaround until the next stable version is published if this can be solved.

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -1,5 +1,5 @@
 pool:
-    vmImage: 'macos-latest'
+    vmImage: 'macOS-12'
 steps:
   - task: UseDotNet@2
     displayName: 'Use .NET 6.0.400 SDK'

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -2,10 +2,10 @@ pool:
     vmImage: 'macos-latest'
 steps:
   - task: UseDotNet@2
-    displayName: 'Use .NET 6.0.200 SDK'
+    displayName: 'Use .NET 6.0.400 SDK'
     inputs:
       packageType: 'sdk'
-      version: 6.0.202
+      version: 6.0.400
   - task: DotNetCoreCLI@2
     displayName: 'Pack Templates'
     inputs:

--- a/templates/csharp/xplat/global.json
+++ b/templates/csharp/xplat/global.json
@@ -1,7 +1,7 @@
 {
     "sdk": {
-        "version": "6.0.202",
-        "rollForward": "latestFeature"
+        "version": "6.0.400",
+        "rollForward": "disable"
     },
     "msbuild-sdks": {
         "Microsoft.Build.Traversal": "1.0.43",

